dispatch/static/manager/src/js/actions/PagesActions.js

```diff
@@ -0,0 +1,177 @@
+import R from 'ramda'
+import { normalize, arrayOf } from 'normalizr'
+import { push } from 'react-router-redux'
+
+import * as types from '../constants/ActionTypes'
+import { pageSchema } from '../constants/Schemas'
+import DispatchAPI from '../api/dispatch'
+
+import ContentStateHelper from '../components/ContentEditor/ContentStateHelper'
+
+function preparePayload(data) {
+
+  if (data._content) {
+    // Convert page contentState to JSON array
+    data.content = ContentStateHelper.toJSON(data._content)
+    delete data._content
+  }
+
+  // Set template_id
+  data.template_id = data.template
+  delete data.template
+
+  if (R.has('featured_image', data) && data.featured_image) {
```

Comment:
why not just
```js
if (data.featured_image)
```
? it's only one key level, not nested, so if it is missing or falsey you'll just get `undefined` and fail the if anyway.